front-end/src/pages/ProposalEdit/ProposalEdit.js

```diff
@@ -0,0 +1,648 @@
+        checkedG: false,
+        checkedH: false,
+    });
+    const [valueBudget, setValueBudget] = useState();
+    const [state, setState] = useState({
+        type: '',
+        status: ''
+    });
+    const [description, setDescription] = useState({
+        content: "Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna wirl Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna wirl Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna wirl Lorem ipsum dolor sit amet, consectetur adipisicing rl Lorem ipsum Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna wirl Lorem ipsum dolor sit ame"
+    });
+    const [hasErrorType, setHasError] = useState(false)
+    const [hasErrorStatus, setHasErrorStatus] = useState(false)
+    const [hasErrorCateory, setHasErrorCategory] = useState(false)
+    const [hasErrorDescription, setHasErrorDescription] = useState(false)
+    const [hasErrorTitle, setHasErrorTitle] = useState(false)
+    const [title, setTitle] = useState("")
+    const [showHistory, setShowHistory] = useState(true)
+    const { errors, handleSubmit } = useForm({
+        criteriaMode: "all"
+    });
+    const onSubmit = data => {
+        // Title validation
+        if (title == "") {
+            setHasErrorTitle(true);
+        } else {
+            setHasErrorTitle(false)
+        }
+        // Type validation
+        if (state.type == "") {
+            setHasError(true);
+        } else {
+            setHasError(false)
+        }
+        // Categories validation
+        for (const [key, value] of Object.entries(stateCheckBox)) {
+            if (value == true) {
+                setHasErrorCategory(false)
+                break;
+            } else {
+                setHasErrorCategory(true)
+            }
+        }
+        // Description validation
+        if (description.content.length >= 100 && description.content.length <= 580) {
+            setHasErrorDescription(false)
+        } else {
+            setHasErrorDescription(true)
+        }
+
+    };
+
+    const handleChange = (event) => {
+        const name = event.target.name;
+        if (event.target.value == "") {
+            setHasError(true)
+        } else {
+            setHasError(false)
+        }
+        setState({
+            ...state,
+            [name]: event.target.value,
+        });
+    };
+
+    const handleChangeStatus = (event) => {
+        const name = event.target.name;
+        if (event.target.value == "") {
+            setHasErrorStatus(true)
+        } else {
+            setHasErrorStatus(false)
+        }
+        setState({
+            ...state,
+            [name]: event.target.value,
+        });
+    };
+
+    const handleChangeCheckboxs = (event) => {
```

Comment:
This validation seems not working as it should. 
If you observe, error message is displaying even if we uncheck one of the checkbox (eventhough there are other checkboxes already checked)

In the below image it shouldn't show error since one checkbox is already checked 
![image](https://user-images.githubusercontent.com/7323026/97217743-033a9100-17c8-11eb-9cff-65260aafc955.png)